ext/standard/tests/mail/mail_basic7.phpt

```diff
@@ -135,71 +148,14 @@ A Message
 
 ************* TEST ******************
 -- All Mail Content Parameters --
-
-Warning: mail(): 'orig-date' header must be at most one header. Array is passed for 'orig-date' in %s on line %d
-
-Warning: mail(): 'from' header must be at most one header. Array is passed for 'from' in %s on line %d
-
-Warning: mail(): 'sender' header must be at most one header. Array is passed for 'sender' in %s on line %d
-
-Warning: mail(): 'reply-to' header must be at most one header. Array is passed for 'reply-to' in %s on line %d
-
-Warning: mail(): Extra header cannot contain 'To' header in %s on line %d
-
-Warning: mail(): 'bcc' header must be at most one header. Array is passed for 'bcc' in %s on line %d
-
-Warning: mail(): 'message-id' header must be at most one header. Array is passed for 'message-id' in %s on line %d
-
-Warning: mail(): 'in-reply-to' header must be at most one header. Array is passed for 'in-reply-to' in %s on line %d
-bool(true)
-To: user@example.com
-Subject: Test Subject
-
-A Message
+TypeError: Header "orig-date" must be of type string, array given
```

Comment:
We're losing test coverage here, as previously multiple errors were checked with one call...